### PR TITLE
Create To-do app

### PR DESCRIPTION
**Pull request by ekenecf**
Created a basic to-do app using 
1. HTML
2. CSS 
3. JavaScript

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const render = ` <div class="books-view"> <p>${book.author} by ${book.title}</p> <button id='${index}' class='remove'>Remove</button> </div>`;
+    newLine.innerHTML += render;
+  });
+}
+renderBooks();
+function remBook(index) {
+  books = books.filter((bk, i) => index !== i);
+  localStorage.setItem('bookDetail', JSON.stringify(books));
+  window.location.reload();
+}
```
Comment (keroloussamy) [inline index.js]: - [ ] Kindly make these methods inside your class to match the requirements `Create class methods to add and remove books.`.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ekenecf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add `.gitignore` file and add to it `node_modules` to not upload all node packages to githup.



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (ekenecf) [reply index.js]: Hello @keroloussamy. As requested by you, I have made those methods inside my class to match the requirements

**Review by dasileker — APPROVED** [review]
### **Hi @ekenecf,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**